tsconfig.json

```diff
@@ -1,48 +1,26 @@
 {
   "compilerOptions": {
+    "target": "ES2020",
     "outDir": "dist",
-    "module": "esnext",
-    "target": "es6",
-    "lib": ["es5", "es6", "es7", "es2017", "dom"],
+    "rootDir": "./",
+    "baseUrl": "./src",
+    "lib": ["ES2020", "DOM", "DOM.Iterable", "ESNext"],
+    "moduleResolution": "node",
     "sourceMap": true,
-    "skipLibCheck": true,
-    "allowJs": true,
-    "jsx": "react-jsx",
     "esModuleInterop": true,
-    "emitDeclarationOnly": true,
-    "moduleResolution": "node",
     "resolveJsonModule": true,
-    "strict": true,
-    "rootDir": "./src",
-    "baseUrl": "./src",
-    "forceConsistentCasingInFileNames": true,
-    "noImplicitReturns": true,
-    "noImplicitThis": true,
-    "noImplicitAny": true,
-    "strictNullChecks": true,
-    "suppressImplicitAnyIndexErrors": true,
-    "noUnusedLocals": false,
-    "declaration": true,
-    "declarationDir": "dist/types",
-    "allowSyntheticDefaultImports": true,
-    "experimentalDecorators": true,
-    "types": ["@emotion/react/types/css-prop", "node", "jest", "@testing-library/jest-dom"],
+    "noEmit": false,
+    "jsx": "react-jsx",
+    "module": "ESNext",
+    "types": ["@emotion/react/types/css-prop"],
+    "allowUnreachableCode": false,
+    /* Linting */
+    //    "strict": true,
+    //    "noUnusedLocals": true,
+    //    "noUnusedParameters": true,
```

Comment:
comments

Reply:
i suggest leaving them and reopening them when we finish these changes.